### PR TITLE
Refactor/Sync method names and commentaries to it

### PR DESCRIPTION
**Pull request by carpawell**
Rearrange/Fix/Sync commentaries to public methods or functions.

### CODE REVIEW

File: pkg/local_object_storage/blobstor/info.go [hunk 1]
```diff
@@ -2,7 +2,7 @@ package blobstor
 
 import "github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/fstree"
 
-// FSTree returns file-system tree for big object store.
+// DumpInfo returns file-system tree for big object store.
```
Comment (alexvanin) [inline pkg/local_object_storage/blobstor/info.go:5]:
```suggestion
// DumpInfo returns information about blob stor.
```

File: pkg/local_object_storage/blobovnicza/get_range.go [hunk 2]
```diff
@@ -34,7 +34,7 @@ func (p *GetRangeRes) RangeData() []byte {
 	return p.rngData
 }
 
-// Get reads the object from Blobovnicza by address.
+// GetRange reads the object from Blobovnicza by address.
```
Comment (alexvanin) [inline pkg/local_object_storage/blobovnicza/get_range.go:37]:
```suggestion
// GetRange reads range of the object from Blobovnicza by address.
```

File: pkg/innerring/processors/governance/processor.go [hunk 3]
```diff
@@ -15,9 +15,9 @@ import (
 	"go.uber.org/zap"
 )
 
-// GovernanceProcessor manages governance sync tasks. This process must not be
-// interrupted by other sync operation, so we limit pool size for processor to
-// one.
+// ProcessorPoolSize limits pool size for audit Processor. Processor manages
```
Comment (alexvanin) [inline pkg/innerring/processors/governance/processor.go:18]:
```suggestion
// ProcessorPoolSize limits pool size for governance Processor. Processor manages
```

File: pkg/local_object_storage/shard/shard.go [hunk 4]
```diff
@@ -110,7 +110,7 @@ func WithMetaBaseOptions(opts ...meta.Option) Option {
 	}
 }
 
-// WithMetaBaseOptions returns option to set internal metabase options.
+// WithWriteCacheOptions returns option to set internal metabase options.
```
Comment (alexvanin) [inline pkg/local_object_storage/shard/shard.go:113]:
```suggestion
// WithWriteCacheOptions returns option to set internal write cache options.
```

File: pkg/morph/client/netmap/config.go [hunk 5]
```diff
@@ -13,7 +13,7 @@ type ConfigArgs struct {
 	key []byte
 }
 
-// EpochValues groups the stack parameters
+// ConfigValues groups the stack parameters
 // returned by get epoch number test invoke.
```
Comment (alexvanin) [inline pkg/morph/client/netmap/config.go:17]:
```suggestion
// returned by get config test invoke.
```

**Review by alexvanin — CHANGES_REQUESTED** [review]

File: pkg/innerring/processors/governance/processor.go (hunk 3 above)
Comment (carpawell) [reply pkg/innerring/processors/governance/processor.go:18]: fixed

File: pkg/local_object_storage/blobovnicza/get_range.go (hunk 2 above)
Comment (carpawell) [reply pkg/local_object_storage/blobovnicza/get_range.go:37]: fixed

File: pkg/local_object_storage/blobstor/info.go (hunk 1 above)
Comment (carpawell) [reply pkg/local_object_storage/blobstor/info.go:5]: fixed

File: pkg/local_object_storage/shard/shard.go (hunk 4 above)
Comment (carpawell) [reply pkg/local_object_storage/shard/shard.go:113]: fixed

File: pkg/morph/client/netmap/config.go (hunk 5 above)
Comment (carpawell) [reply pkg/morph/client/netmap/config.go:17]: fixed

**Review by alexvanin — APPROVED** [review]

**Pull request merged**